fi.skyjake.Lagrange.json

```diff
@@ -0,0 +1,35 @@
+{
+    "app-id": "fi.skyjake.Lagrange",
+    "default-branch": "release",
```

Comment:
this shouldn't be in here for a flathub build - everything in flathub is on the "stable" branch.

Reply:
AFAIK `default-branch` is the correct thing if someone wants a sane default for local builds – this gets overwritten as opposed to just `branch` which is not.

Reply:
Hrm, yeah - that's a point. I've always thought that the ergonomics of switching branches are slightly sub-optimal and observe that most local testing seems to happen with different repos (e.g. flathub in system and changes in --user) but yeah, you've got a point.